test/coffeescript-preview.test.ts

```diff
@@ -51,6 +52,11 @@ describe("CoffeeScriptPreview Tests", () => {
     it("updateContent() will call the content provider to update content", ()=>{
         const stub = sinon.stub(this.provider, "updateContent");
         this.cspreview.updateContent("new_file.coffee");
+        let separator = "//";
+        if (os.platform() === "win32") {
+            separator = "\\";
+        }
+        assert.equal(stub.getCall(0).args[0].toString().includes(separator), true);
```

Comment:
This looks like a great start. I'm thinking we should divide the test into two and test both scenarios (Windows and non-Windows). I'll go ahead and merge as is and see about adding it in.